### PR TITLE
Log index test

### PR DESCRIPTION
**Pull request by kjazgar**
Fixes | Closes | Resolves #

## Changes:
Added test that checks if log indexes are unique according to https://github.com/NethermindEth/nethermind/pull/2996.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Other (please describe): test

### CODE REVIEW

File: src/Nethermind/Nethermind.JsonRpc/Modules/Parity/ParityRpcModule.cs [hunk 1]
```diff
@@ -86,7 +86,21 @@ public ResultWrapper<ReceiptForRpc[]> parity_getBlockReceipts(BlockParameter blo
...
+        {
+            int logIndexInBlock = 0;
+            for (int receiptIndex = 0; receiptIndex < resultAsArray.Length; receiptIndex++)
+            {
+                for (int logIndexInTx = 0; logIndexInTx < resultAsArray[receiptIndex].Logs.Length; logIndexInTx++)
+                {
+                    resultAsArray[receiptIndex].Logs[logIndexInTx].LogIndex = logIndexInBlock++;
+                }
+            }
         }
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.JsonRpc/Modules/Parity/ParityRpcModule.cs:104]: We need to check if we don't need this in other RPC methods too

Comment (MarekM25) [reply src/Nethermind/Nethermind.JsonRpc/Modules/Parity/ParityRpcModule.cs:104]: @kjazgar will recheck the rest of endpoints, but for now it looks good for me.

**Review by MarekM25 — COMMENTED** [review]
@kjazgar Could you revert src/tests? Then we can merge it

**Review by MarekM25 — APPROVED** [review]

File: src/Nethermind/Nethermind.JsonRpc/Modules/Parity/ParityRpcModule.cs (hunk 1 above)
Comment (LukaszRozmej) [reply src/Nethermind/Nethermind.JsonRpc/Modules/Parity/ParityRpcModule.cs:104]: @kjazgar @MarekM25 can you check eth_getTransactionReceipt?

**Pull request merged**